curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f9cb678070ca1668898c70.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 65f9cb678070ca1668898c70
+title: Step 44
+challengeType: 20
+dashedName: step-44
+---
+
+# --description--
+
+In the same way `__add__` is called under the hood when two objects are added together, the `__sub__` method is called implicitly in case of subtraction.
```

Comment:
```suggestion
In the same way that `__add__` is called under the hood when two objects are added together, the `__sub__` method is called implicitly in case of subtraction.
```